final-project/static/scripts.js

```diff
@@ -8,6 +8,18 @@ $(function() {
 
 });
 
+/**
+ * Edit product information.
+ */
+function editProduct()
+{
```

Comment:
nit: Hmmmm, this style is not common in Javascript. It's more C, and its better to be consistent w/ mainstream standards.

Reply:
When you say this style, are you referring to only the comment or the function as well?

In the Code Academy tutorial, it says `//` is for single line comments, and `/*  */` is for multi-line. Are you saying to only use the `//` syntax?  

If your answer is yes, it would make life easier. Cloud9 defaults to the single-line comment syntax, regardless of the number of lines, so I could eliminate a few keystrokes haha

Reply:
```javascript
// Edit product information.
function editProduct() {
  // ...
}
```